api/types/src/transaction.rs

```diff
@@ -869,6 +870,7 @@ pub enum TransactionSignature {
     MultiEd25519Signature(MultiEd25519Signature),
     MultiAgentSignature(MultiAgentSignature),
     FeePayerSignature(FeePayerSignature),
+    Secp256k1Signature(Secp256k1Signature),
```

Comment:
Can you please accurately name this `Secp256k1Ecdsa` signature? It's currently incomplete and confusing. Plus, we could have all sorts of other signatures over the `secp256k1` curve later on.

Reply:
what other signature types are there?

Reply:
EC-Schnorr, for example. `Secp256k1` sounds more a curve to me, rather than a signature scheme.